.rubocop.yml

```diff
@@ -1,3 +1,6 @@
+AllCops:
+  TargetRubyVersion: 2.2
```

Comment:
needs to be the minimum version supported or it will recommend things that are not possible

Reply:
I don't know exactly what is the actual minimum version supported. I developed on 2.5, and I tried to  avoid too modern syntax, but I am not sure I missed something and what the min version would be. Is there any quick-and-easy way to check that?

Reply:
oh... just saw #30